pages/table-fragments/grid-navigation-custom.page.tsx

```diff
@@ -284,21 +289,22 @@ export default function Page() {
                               }
                             }}
                           />
-                        </th>
+                        </Cell>
                       ))}
                     </tr>
                   </thead>
                   <tbody>
                     {sortedItems.map((item, rowIndex) => (
                       <tr key={item.id} {...getTableRowRoleProps({ tableRole, rowIndex, firstIndex: 0 })}>
                         {visibleColumnDefinitions.map((column, colIndex) => (
-                          <td
+                          <Cell
```

Comment:
[Before these changes](https://d21d5uik3ws71m.cloudfront.net/components/935524f97ebab6c43351cb269c7945f1c5ce4ffb/dev-pages/index.html#/light/table-fragments/grid-navigation-custom) the select and input fields in the "Select" and "Input" columns could be accessed directly with the grid navigation, and [now](https://d21d5uik3ws71m.cloudfront.net/components/87eabd0a9d0d6855cea278e4c4aacf423ac762cc/dev-pages/index.html#/light/table-fragments/grid-navigation-custom) the entire cell is focused instead and one further tab is necessary to access these inner interactive elements. Is this intended?

Reply:
The select and input components are not yet registered with single tab focus navigation context, that's why the focusing works differently.